client-datastore-common-test/src/main/java/com/telecominfraproject/wlan/client/datastore/BaseClientDatastoreTest.java

```diff
@@ -1036,8 +1036,8 @@ public void testClientSessionPagination()
        assertEquals(expectedPage2Eq_1or2Strings, actualPage2Loc_1or2Strings);       
 
        //test the results for ( location_1 or  location_2 ) and equipment_1only
-       page1Eq_1 = testInterface.getSessionsForCustomer(customerId_1, new HashSet<Long>(Arrays.asList(equipmentId_1)), new HashSet<Long>(Arrays.asList(locationId_1, locationId_2)), Collections.emptyList(), context);
-       page2Eq_1 = testInterface.getSessionsForCustomer(customerId_1, new HashSet<Long>(Arrays.asList(equipmentId_1)), new HashSet<Long>(Arrays.asList(locationId_1, locationId_2)), Collections.emptyList(), page1Eq_1.getContext());
+       page1Eq_1 = testInterface.getSessionsForCustomer(customerId_1, new HashSet<Long>(Arrays.asList(equipmentId_1)), new HashSet<Long>(Arrays.asList(locationId_1, locationId_2)), null, Collections.emptyList(), context);
+       page2Eq_1 = testInterface.getSessionsForCustomer(customerId_1, new HashSet<Long>(Arrays.asList(equipmentId_1)), new HashSet<Long>(Arrays.asList(locationId_1, locationId_2)), null, Collections.emptyList(), page1Eq_1.getContext());
```

Comment:
where are the tests for macSubstring that is not null ?

Reply:
Hm, looks like my commit to the branch missed some files. Not sure how that happened, I will resolve that.